packages/features/portfolio/src/containers/portfolio-overview/containers/note-holdings-overview.tsx

```diff
@@ -0,0 +1,63 @@
+import { Box, styled, Typography, useTheme } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+import PositionCard from '../components/position-card';
+import { usePortfolioNOTETable } from '@notional-finance/portfolio-feature-shell/hooks';
+
+interface IProps {
+  data: ReturnType<typeof usePortfolioNOTETable>['noteData'];
+}
+
+const NOTEHoldingsOverview = ({ data }: IProps) => {
+  const theme = useTheme();
```

Comment:
This is unused. You should be seeing these errors in the linter when you commit now. All the typecheck and lint hooks should work now so no need to use --no-verify anymore